### PR TITLE
SONARIAC-537 Remove Tree keyword from existing Tree elements and add …

### PR DESCRIPTION
**Pull request by mstachniuk**
…Instruction keyword for relevant element

### CODE REVIEW

File: iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/AddInstruction.java [hunk 1]
```diff
@@ -19,5 +19,5 @@
  */
 package org.sonar.iac.docker.tree.api;
 
-public interface HereDocumentTree extends LiteralListTree {
+public interface AddInstruction extends TransferInstruction {
```
Comment (rudy-regazzoni-sonarsource) [inline iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/AddInstruction.java:22]: To double check, I guess it's a diff issue from Github

File: iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/NoneInstruction.java [hunk 2]
```diff
@@ -19,5 +19,5 @@
  */
 package org.sonar.iac.docker.tree.api;
 
-public interface ShellFormTree extends LiteralListTree {
+public interface NoneInstruction extends Instruction {
```
Comment (rudy-regazzoni-sonarsource) [inline iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/NoneInstruction.java:22]: Now that I see it, not sure if None should be an Instruction.
Anyway, not in the scope of this ticket, I will note it to check this later.

**Review by rudy-regazzoni-sonarsource — APPROVED** [review]
LGTM ! I just added a general comment not related to this change directly and not in the scope of this ticket, you can merge whenever you want ;)

File: iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/NoneInstruction.java (hunk 2 above)
Comment (mstachniuk) [reply iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/NoneInstruction.java:22]: I think that git wrongly detects the move of the file.

**Pull request merged**